### ISSUE TITLE
VR Documentation - Accounting - Consolidate - Rig Build

### ISSUE THREAD

**Issue by JayeHarrill** [DecentralVan/decentral#179]
- Consolidate info from Slack & Sources
- Move to Wiki/documentation stored in the GitHub
- Consolidate Accounting
- Get people to pay
- Document Parts List documentation + status
- Status Report

**Comment by JayeHarrill** [comment]
https://github.com/DecentralVan/decentral/issues/173#issuecomment-282124759

**Comment by JayeHarrill** [comment]
Slack History from start-date to Mar 13 2017.
- Use for consolidation, placed here temporarily to avoid deletion

https://docs.google.com/document/d/1IWe9NkVIK0HWJO-OE2jAsLzc7NRYPGPblxs65gMw3Dk/edit?usp=sharing

**Comment by JayeHarrill** [comment]
Updated Issue 
-Consolidate info from Slack & Sources (Recipts, chat log, who's comited, who's paid, what parts are bought, what parts are needed) - accounting / Status report
    - Consolidate Accounting
    - Document Parts List documentation + status
    - Status Report

- Move to Wiki/documentation stored in the GitHub - "postmortem" write up + basic documentation

- Get people to pay

**Comment by JayeHarrill** [comment]
Accounting + parts list, DRAFT

https://docs.google.com/spreadsheets/d/1sy1bLzm9lyS48v7CrUtuoLpP64UZaGNMTyJcPjIRwX4/edit?usp=sharing

**Comment by JayeHarrill** [comment]
Still to do
- Pay for shipping upon arrival of Mobo (Jared)
- Pay Jared, Taylor, Kiko back the difference + Update commons fund accounting
- Talk to Manuel, Chelsea (Check Slack for comments on status)
- Buy RAM
- Update + Fix legibility of VR Accounting (Consider Multi-page?) 
- Move to Wiki/documentation stored in the GitHub - "postmortem" write up + basic documentation
- Build + Test Rig

**Comment by JayeHarrill** [comment]
Accounting has been updated. Still a Draft, so a bit messy.
TLDR,
- Need to still buy RAM, budget approx $200
- Shipping cost of Mobo to pay upon pick up, estimate $15
- Mobo should arrive Marc 20th at the latest
- If we count the NVMe card Kiko donated (he still has first right of refusal, as he "owns" it) we are under budget by about $106. But if you don't count the card, we are over budget by about $95
- Total approx cost $2142.71 (Rig + VR)
- Total needed to crowd-fund $1071.36
- Total crowdfunded $976 + $201.59 (Value of NVMe card)
- Approx date of operational status April 3th-7th

**Comment by JayeHarrill** [comment]
Kenta is all paid up

**Comment by ghost** [comment]
Great success, let's finalise how much the fiat purchasers are owed and get this closed!!

I purchased the Vive was 1150 (minus my 75 contribution). I also purchased the windows 10 for $150 but I think that was beyond the scope of our budget? I bet we could find some more crowd fund participants especially if we bestow some access privileges to the funders?

**Comment by JayeHarrill** [comment]
Indeed! Well done everyone.
I actually have done most of the accounting in a shared doc earlier up. Just a few more bits needs to be filled. Then we can pay out to those owed.

Can confirm Chelsea paid into the general address.

**Comment by Latrasis** [comment]
Could we include the games I bought as a small contribution? It was approx $40.

Since I received Jared's money - $20. I'm willing to contribute $150.00

**Comment by JayeHarrill** [comment]
Well the $20 was for the games I wanted. But I think we should create a separate spreadsheet for games. Seperate budget, separate donation.
